src/views/Login.vue

```diff
@@ -75,7 +76,7 @@ export default {
       console.log(localStorage.getItem('access_token'))
     }
   },
-  async created () {
+  async mounted () {
```

Comment:
Code comments would be nice. What is "mounted"?

Reply:
"created" happens before "mounted" and doesn't wait for async methods to complete. https://vuejs.org/v2/guide/instance.html#Lifecycle-Diagram